roles/rabbitmq/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# A comma separated list of plugins to enable (existing plugins will not be disabled)
+rabbitmq_plugins: ''
```

Comment:
I'm +1 for a list as it was before (to follow the yaml data types), we just need to transform it afterwards in `rabbitmq_plugin`.

Also what will happen if on first deployment we have `rabbitmq_plugins: [rabbitmq_management]` and on second deployment we remove it: `rabbitmq_plugins: []`.
Will that automatically disable unneeded plugins for us?

Reply:
OK. I will turn it into a yaml list, and then join(',') it in the play.